src/docs.md

```diff
@@ -378,6 +378,15 @@ To invalidate caches and reload:
 cadenzaClient.reload({ invalidateCaches: true });
 ```
 
+### Close itself
```

Comment:
I think we can be more clear here. Proposal:
> Close the Cadenza Dialog or Window That Embeds the Custom Application

Also note: Headings in technical documentation should be in title case: https://conf.disy.net/display/EHB/AsciiDoc+Documentation+Practices#AsciiDocDocumentationPractices-TitleCase